src/plugins/recordTypes/person/optionLists.js

```diff
@@ -215,100 +215,4 @@ export default {
       },
     }),
   },
-  genders: {
```

Comment:
Let's leave this in for now, until a future release where we can convert all the static lists.